examples/proxy/record-encryption/README.md

```diff
@@ -1,20 +1,17 @@
 # Streams for Apache Kafka Proxy Record Encryption Filter Examples
 
-The Record Encryption filter provides an Encryption-at-Rest solution for Apache Kafka(tm) which is transparent to both clients and brokers. The filter takes
-responsibilty to encrypt messages that are sent by producing applications so the Kafka Broker never sees the plain text content of the messages.  The filter also takes responsibilty
-to decrypt the message before they are returned to consuming applications.
+The Record Encryption filter provides an Encryption-at-Rest solution for Apache Kafka(tm) which is transparent to both clients and brokers. 
+The filter is responsible for encrypting messages that are sent by producing applications so the Kafka Broker never sees the plain text content of the messages.  
+The filter also decrypts messages before they are returned to consuming applications.
 
-In this directory, you'll find examples that help you deploy AMQ Stream Proxy with the Record Encryption Filter to your OpenShift Cluster so that you my try out the feature together
+In this directory, you'll find examples that help you deploy Streams for Apache Kafka Proxy with the Record Encryption filter to your OpenShift Cluster so that you may try out the feature together
 with you own application.
 
-The filter relies on a Key Management System (KMS). The role of the KMS is to provide cryptographic functions and act as a repository of key-material. The KMS is *not* part of Streams for Apache Kafka.  It is external and must be provided by the deployer of the system.  For this Tech Preview release, only HashiCorp Vault&#8482; is supported.
+The filter relies on a Key Management System (KMS). The role of the KMS is to provide cryptographic functions and act as a repository of key-material. The KMS is *not* part of Streams for Apache Kafka.  It is external and must be provided by the deployer of the system.  For this Technology Preview release, you can choose
```

Comment:
`the deployer of the system` could be read as requiring a dedicated KMS, as the person running the example is the deployer of the system, but we could use the platform provided KMS.

Reply:
@PaulRMellor I'm inclined just to delete the sentence "It is external and must be provided by the deployer of the system".  It is repetitious. The previous sentence tells them the KMS is not part of streams.  The follow sentence present the choice.